### ISSUE TITLE
[Keyboard Navigation - Logic apps - Design]: Focus order is not logical while navigating the controls under "Design" section.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6098]
### Describe the issue 

User Experience: 

 Keyboard dependent users who navigate the application sequentially and expect the focus order to be consistent with the sequential reading order will face difficulty if keyboard focus order is not aligned with either the meaning or operation of the UI. 

 Note: User credentials should NOT be included in the bug.

Pre-requisites:

- Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed. 
- Navigate to Azure control present on the right side and select it. 
- Navigate to LogicApp icon present under workspace section, click on tat and select 'Create New Project'. 
- Navigate select stateful workflow, then enter the name of the workflow. 
- Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter. 
- Navigate to 'Open in current window' and select it. 
- A dialog will appear, select 'Yes', now the workflow will open in new window. 

Repro Steps:

- Open Visual Studio app. 
- Navigate to any workflow and right click on it, then select 'open designer'. 
- Navigate all the controls of the designer screen. 
- Observe whether Focus order is not logical while navigating the controls under "Design" section or not. 

Actual Result: 

 Focus order is not logical while navigating the controls under "Design" section.

 Current Order: Split Editor right button > status bar(at bottom) > Https button > Insert button > Create button > Insert button > Error (Header cotrols) > Connection required graphic > Toolbar Zoom in button.

Expected result: 

 Focus order should be logical while navigating the controls under "Design" section.

 Expected Order: Split Editor right button > Error (Header controls) > Https button > Insert button > Create button > Connection required graphic > Insert button > Toolbar controls >status bar(at bottom).

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
@rllyy97  can you take a look at this one?
